test/e2e/kubectl.go

```diff
@@ -17,17 +17,13 @@ limitations under the License.
 package e2e
 
 import (
-	"bytes"
 	"encoding/json"
-	"fmt"
-	"os/exec"
+	"errors"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
```

Comment:
Move this import into it's own import group below the standard imports and above the ginkgo import (https://github.com/golang/go/wiki/CodeReviewComments#imports).